origin-dapp/src/components/form-widgets/photo-picker.js

```diff
@@ -87,13 +87,13 @@ class PhotoPicker extends Component {
   }
 
   render() {
-    const { schema, required } = this.props
+    const { schema, required, value } = this.props
     const {
-      pictures,
       showMaxImageCountMsg,
       showCropModal,
       imageFileObj
     } = this.state
+    const pictures = value
```

Comment:
This will conflict with some changes in https://github.com/OriginProtocol/origin/pull/882
We can see which PR gets merged first and sort it out later